### ISSUE TITLE
Not working for other platforms

### ISSUE THREAD

**Issue by mikaelhogstrom** [dantogno/UnityAzureSample#1]
First off, thanks for sharing! 

I tried to build this for other platforms (Android, UWP and Web GL) but could not get it to work. I tried getting the dll versions for e.g. monoandroid, portable etc as well as just using the .net45 versions from nuget but still didn't work. Might be that I somehow didn't get the correct combination though. 

Worth noting is that the Android version actually built but didn't get any data, while UWP and Web GL did not build at all.

If you could update the repo or just comment with some pointers that would be great!

**Comment by dantogno** [comment]
Yeah, there are a variety of issues blocking different platforms, all stemming from the 4.6 update being "experimental." In general, I think these issues will gradually be resolved, but the timetable is uncertain as it depends on Unity and how they choose to move forward with the scripting upgrade, etc.

My understanding is that there is a bug in the Unity version of Mono that blocks Mac. This has been fixed in Mono and so I expect it to get fixed in Unity at some point. 

There is a known Unity bug blocking Android that you can track here: https://issuetracker.unity3d.com/issues/android-dot-net-4-dot-6-httpclient-dot-getasync-fails-to-get-address-with-unable-to-find-libc-error

Web GL doesn't support threading afaik so I don't think the Azure Mobile SDK will work for it ever.

And to be honest, I haven't tried it on iOS. I expect I will at some point :)

**Comment by mikaelhogstrom** [comment]
So basically just wait and hope for the best I guess. For now I'll stick to using REST I guess.
